### PR TITLE
Register modal

### PR DESCRIPTION
**Pull request by BarbaraBochinska**
Added modal to registration with inputs and validation

### CODE REVIEW

**Comment by sznowak13** [comment]
Please, review my changes!

File: front/src/sideBard/LoggedOutSidebar.js [hunk 1]
```diff
@@ -25,36 +32,32 @@ export default class LoggedOutSidebar extends Component {
...
+                  <FormGroup>
+                    <ButtonToolbar>
+                      <Button variant="outline-secondary" block>
+                        Submit
+                      </Button>
+                    </ButtonToolbar>
+                  </FormGroup>
+                </Form>
+              </Dropdown.Item>
+            </Dropdown>
```
Comment (sznowak13) [inline front/src/sideBard/LoggedOutSidebar.js]: I wonder how this formatting issues came up so often.

File: front/src/sideBard/RegisterModal.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    .isRequired("Email is required"),
+  email2: StringType().addRule(
+    (value, data) => value === data.email1,
+    "Emails must match"
+  ),
+  password1: StringType().isRequired("Password is required"),
+  password2: StringType().addRule(
+    (value, data) => value === data.password1,
+    "Passwords must match"
+  )
```
Comment (sznowak13) [inline front/src/sideBard/RegisterModal.js]: I really like this approach of the validation that rsuite gives us. Please, read about it more in the docs if You are confused. https://rsuitejs.com/components/schema

File: front/src/sideBard/RegisterModal.js [hunk 3]
```diff
@@ -67,11 +67,11 @@ export default class RegisterModal extends Component {
               </FormGroup>
               <FormGroup>
                 <ControlLabel>Password</ControlLabel>
-                <FormControl name="password1" />
+                <FormControl name="password1" type="password" />
```
Comment (BarbaraBochinska) [inline front/src/sideBard/RegisterModal.js]: easy

File: front/src/sideBard/RegisterModal.js [hunk 4]
```diff
@@ -39,7 +39,7 @@ export default class RegisterModal extends Component {
     const { formData } = this.state;
     return (
       <div>
-        <Modal show={this.state.show} onHide={this.close} size="xs">
+        <Modal show={this.state.show} onHide={this.close} size="lg">
```
Comment (BarbaraBochinska) [inline front/src/sideBard/RegisterModal.js]: also easy good job

**Pull request merged**